src/main/java/manageezpz/model/AddressBook.java

```diff
@@ -44,6 +51,100 @@ public AddressBook(ReadOnlyAddressBook toBeCopied) {
         resetData(toBeCopied);
     }
 
+    /**
+     * Gets the list of all tasks.
+     * @return A string representing all tasks
+     */
+    public String listTask() {
```

Comment:
Hey @Denniszedead, can you check if you have handled the case whereby there is 0 task in the list? I'm getting a NoSuchElementException when using list command when there is no task

Reply:
![image](https://user-images.githubusercontent.com/70746290/158432769-e8411d32-45a6-4f03-9eda-0e712bc0db88.png)